timetracker/timetracker/urls.py

```diff
@@ -19,7 +19,20 @@
 
 
 def hello_world(request):
-    return HttpResponse('Hello World')
+    blank_html='''
+           <html>
+              <body>
+                 <form>
+                   Name:<input type='text' name='name' value=' '>
+                   <input type='submit' value='Hit Me !'>
+                 </form>
+              </body>
+            </html>
+         '''
+    if 'name' in request.GET:
+        return HttpResponse('{}, Welcome to Melbourne Django School'.format(request.GET.get('name')))
```

Comment:
Since you've already tested that 'name' is in request.GET, you didn't need to use .get()